src/main/groovy/uk/co/metadataconsulting/sentinel/ExcelReader.groovy

```diff
@@ -81,53 +75,12 @@ class ExcelReader {
         while (cellIterator.hasNext()) {
             Cell cell = cellIterator.next()
             String cellValue = dataFormatter.formatCellValue(cell)
-            if(!cellValue.isEmpty() && cellValue!="") {
-                cellValues << cellValue;
-            }else{
-                break;
+            if ( !cellValue.isEmpty() && cellValue!="" ) {
+                cellValues << cellValue
+            } else {
+                break
             }
         }
         cellValues
     }
-
-    static String getValue(Row row, Cell cell, List data) {
```

Comment:
Hi - if we remove this (and at line 65) then we will get Excel date type (rather than how it was originally formatted) and Excel numbers which turns  1 into 1.0 etc.

Reply:
`getValue` was not being used